doc/plotting/index.rst

```diff
@@ -501,17 +501,20 @@ Or you can save it to an html file::
 Interactive visualization of statistical map slices
 ---------------------------------------------------
 
-:func:`view_stat_map`: open stat map in a Papaya viewer (https://github.com/rii-mango/Papaya)::
+:func:`view_stat_map`: open stat map in a Brainsprite viewer (https://github.com/simexp/brainsprite.js)::
 
     >>> from nilearn import plotting, datasets     # doctest: +SKIP
-    >>> img = datasets.fetch_localizer_button_task()['tmaps'][0]     # doctest: +SKIP
-    >>> view = plotting.view_stat_map(img, threshold=2, vmax=4)     # doctest: +SKIP
+    >>> img = datasets.fetch_localizer_button_task()['tmap']     # doctest: +SKIP
+    >>> html_view = plotting.view_stat_map(img, threshold=2, vmax=4, cut_coords=[-42, -16, 52], title="Motor contrast")     # doctest: +SKIP
```

Comment:
```suggestion
    >>> html_view = plotting.view_stat_map(img, threshold=2, vmax=4, cut_coords=[-42, -16, 52],
    ...                                     title="Motor contrast")     # doctest: +SKIP
```

Reply:
I can make myself and assignee and do it now.

Reply:
@GaelVaroquaux I applied the change

@kchawla-pi I have added you to https://github.com/SIMEXP/nilearn if you want to push changes directly to the PR.